.github/workflows/build-and-test.yaml

```diff
@@ -21,15 +21,25 @@ on:
   push:
     branches:
       - main
-    paths:
-      - '!**.md'
 
 jobs:
   build:
     runs-on: ubuntu-18.04
     timeout-minutes: 90
+    strategy:
+      matrix:
+        go-version: [1.14, 1.15]
+      fail-fast: true
     steps:
-      - uses: actions/checkout@v2
+      - name: Set up Go ${{ matrix.go-version }}
+        uses: actions/setup-go@v1
```

Comment:
In the new project, my preference is we should opt to [setup-go@v2](https://github.com/actions/setup-go#v2)

Reply:
Agree. let's use v2.